### ISSUE TITLE
Either allow http or selfsigned certificates for gotify.

### ISSUE THREAD

**Issue by MatthiasGrandl** [containrrr/shoutrrr#65]
Since gotify is often hosted internally, it doesn't make much sense to force SSL.

**Comment by piksel** [comment]
You can add the certificate to the trusted store like this on debian-ish linuxes:
```sh
sudo cp self-signed-ca.crt /usr/local/share/ca-certificates/ && update-ca-certificates
```
Shoutrrr will use the systems CA store to validate the certificates.
Accepting self-signed certificates is generally a bad idea as it would make it possible to spoof https-connections for users that rely on the verification.

That being said, https should probably be made optional as well. A PR would be welcomed!

**Comment by simskij** [comment]
Im not sure if I agree that it should be optional.

IF made optional, I think SSL should be the default and disabling it should require actively opting out.

**Comment by piksel** [comment]
No, yeah, that's what I meant. I'm not even sure how you would do it "automatically" since it has to do with what URL it sends the API request to.

**Comment by piksel** [comment]
Fixed in #116
